### PR TITLE
Initial client

### PR DESCRIPTION
**Pull request by haiodo**
Initial client and environment variables to use.
Signed-off-by: Andrey Sobolev <haiodo@gmail.com>

### CODE REVIEW

File: .go-header.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
```
Comment (denis-tingajkin) [inline .go-header.yaml:1]: This config is not needed.

File: go.mod [hunk 2]
```diff
@@ -1,3 +1,29 @@
...
+	github.com/pkg/errors v0.9.1
+	github.com/rcrowley/go-metrics v0.0.0-20200313005456-10cdbea86bc0 // indirect
+	github.com/sirupsen/logrus v1.6.0
+	github.com/spiffe/go-spiffe/v2 v2.0.0-alpha.5
+	github.com/stretchr/testify v1.6.1
+	github.com/uber/jaeger-client-go v2.25.0+incompatible // indirect
+	golang.org/x/crypto v0.0.0-20200728195943-123391ffb6de // indirect
+	golang.org/x/net v0.0.0-20200707034311-ab3426394381 // indirect
+	golang.org/x/sys v0.0.0-20200810151505-1b9f1253b3ed // indirect
+	golang.org/x/text v0.3.3 // indirect
```
Comment (denis-tingajkin) [inline go.mod:23]: I feel a few of these indirect can be removed.

Comment (haiodo) [reply go.mod]: Fixed.

File: pkg/config/config.go [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+	}
+	if c.ConnectTo == nil {
+		return errors.New("no NSMGr ConnectTO URL are sepecified")
+	}
+	return nil
+}
+
+// NetworkServiceConfig - defines a network service request configuration
+type NetworkServiceConfig struct {
+	NetworkService string            `default:"" desc:"A name of network service" split_words:"true"`
```
Comment (denis-tingajkin) [inline pkg/config/config.go:65]: Can the client want a few network services?
```suggestion
	NetworkService []string            `default:"" desc:"A name of network service" split_words:"true"`
```

**Pull request merged**